deployments/operator/crd/bases/deviceplugin.intel.com_qatdeviceplugins.yaml

```diff
@@ -60,6 +60,10 @@ spec:
               image:
                 description: Image is a container image with QAT device plugin executable.
                 type: string
+              initImage:
+                description: InitImage is a container image with a script that enables
```

Comment:
this may not be the only function the `initImage` so let rewrite the description to be more generic about device init